src/combo.rs

```diff
@@ -1,11 +1,11 @@
 //! Provides combo support for two keys pressed at once.
 //! E.g. Left + Right arrow at the same time => paste.
-use arraydeque::ArrayDeque;
 use crate::layout::Event;
+use arraydeque::ArrayDeque;
 
```

Comment:
You should put the `heapless` stuff up here to make the code below easier to read (like `layout.rs`):
```rust
use heapless::consts::{U4, U8};
use heapless::Vec;
```

Then stuff like this (line 32):
```rust
heapless::Vec<Event, heapless::consts::U4>
```
...would become this:
```rust
Vec<Event, U4>
```
*So* much easier to read and look at :+1:

Reply:
Yeah, prefer `heapless::Vec` except if you need deque functionnalities.